packages/vue/src/components/organisms/SfFooter/SfFooter.stories.js

```diff
@@ -5,7 +5,9 @@ import {
   SfImage,
   SfButton,
   SfMenuItem,
+  SfInput,
 } from "@storefront-ui/vue";
+import "./SfFooter.stories.scss";
```

Comment:
why? We don't use this scss anymore. if you want any specific css to this story, please add inline styling and remove this scss import. I remove this on purpose. And also we should avoid styling component story specifically too much.